packages/babel-parser/test/fixtures/experimental/module-blocks/valid-trailing-comment-module-blocks/input.js

```diff
@@ -0,0 +1,3 @@
+module {
+  /* leading */ x /* trailing */
```

Comment:
Currently Babel attaches `trailing` as the trailing comment of the Program node because the parser finishes the Program node before `/* trailing */`.

Now the behaviour is aligned to how we parse plain `/* leading */ x /* trailing */`.